http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala

```diff
@@ -764,12 +764,29 @@ class GKEInterpreter[F[_]](
             ).interruptAfter(config.monitorConfig.updateApp.interruptAfter).compile.lastOrError.map(x => x.isDone)
 
           } yield (chartValues, last)
-        case _ => F.raiseError(AppUpdateException(s"App type ${app.appType} not supported on GCP", Some(ctx.traceId)))
+        case _ =>
+          F.raiseError[(String, Boolean)](
+            AppUpdateException(s"App type ${app.appType} not supported on GCP", Some(ctx.traceId))
+          )
       }
 
+      (chartOverrideValues, preUpdateAppOk) <- chartOverridesAndAppOkF
```

Comment:
Why break these into an `=` followed by a `<-`?

Reply:
Is it necessary to flatMap the chartOverridesAndAppOkF object twice to get two distinct ok check functions? Wouldn't it create a totally separate stream function if you did `if (preCheckAppOk)` twice?

Reply:
No, using `=` establishes an `F[_]`, which runs the `isProxyAvailable` twice. Using `<-` in the original block actually prematurely runs the `appOk` check; this was a bug I am fixing here.

Reply:
Gotcha, thanks!